package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "react-native": "^0.57.7"
```

Comment:
why do you need `react-native` as a dev dependency? I don't think this should be included in the library